<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_GetListReturnBooks</class>
 <widget class="QMainWindow" name="form_GetListReturnBooks">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список книг на руках</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbReaderInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string>Читатель</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Читательский билет:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPicTicket">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>241</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/ЧБ.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTicketNumber">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbReaderBio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>551</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>Ваши данные:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbUserActions">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>301</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pbProlong">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Продлить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbReturnBook">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сдать</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbOwnedBooksJournals">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>691</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string>Книги и журналы</string>
    </property>
    <widget class="QTableWidget" name="twOwnedBookJournals">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>651</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView {
    background-color: #fff;
    font-size:13px;
}
QHeaderView::section:horizontal {
    color: #fff;
    border-style: solid;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #4287ff, stop: 1 #356ccc);
 }
QTableView {
    border: 2px solid #3873d9;
    border-top-color: #4287ff;
    border-radius: 4px;
    background-color: #fff;
    gridline-color: #777;
    selection-background-color: #ccdfff;
    color:#333;
    font-size:12px;
 }

QScrollBar:vertical {
     background: #e4e4e4;
     border-top-right-radius: 4px;
     border-bottom-right-radius: 4px;
     width: 12px;
     margin: 0px;
 }
 QScrollBar::handle:vertical {
    background-color: qlineargradient( x1: 0, y1: 0, x2: 1, y2: 0,
                                    stop: 0 #4287ff, stop: 1 #356ccc);
     border-radius: 4px;
     min-height: 20px;
     margin: 0px 2px 0px 2px;
 }
 QScrollBar::add-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Ост.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Место выдачи</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Автор / Журнал</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Заглавие / Год,Номер</string>
      </property>
     </column>
    </widget>
    <widget class="QSlider" name="vsChangeSize">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>50</y>
       <width>23</width>
       <height>521</height>
      </rect>
     </property>
     <property name="maximum">
      <number>566</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="twBookJournalsReturned">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>295</y>
       <width>651</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView {
    background-color: #fff;
    font-size:13px;
}
QHeaderView::section:horizontal {
    color: #fff;
    border-style: solid;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #4287ff, stop: 1 #356ccc);
 }
QTableView {
    border: 2px solid #3873d9;
    border-top-color: #4287ff;
    border-radius: 4px;
    background-color: #fff;
    gridline-color: #777;
    selection-background-color: #ccdfff;
    color:#333;
    font-size:12px;
 }

QScrollBar:vertical {
     background: #e4e4e4;
     border-top-right-radius: 4px;
     border-bottom-right-radius: 4px;
     width: 12px;
     margin: 0px;
 }
 QScrollBar::handle:vertical {
    background-color: qlineargradient( x1: 0, y1: 0, x2: 1, y2: 0,
                                    stop: 0 #4287ff, stop: 1 #356ccc);
     border-radius: 4px;
     min-height: 20px;
     margin: 0px 2px 0px 2px;
 }
 QScrollBar::add-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
     background: none;
     height: 0px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Возврат</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Место выдачи</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Автор / Журнал</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Заглавие / Год,Номер</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbRecomendations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1271</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Новинки и рекомендации на основе вашей взятой литературы</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
